app/jobs/whats_app_adapter/set_webhook_url.rb

```diff
@@ -8,12 +8,13 @@ def perform(organization_id:)
       organization = Organization.find_by(id: organization_id)
       return unless organization && organization.three_sixty_dialog_client_api_key.present?
 
-      base_uri = ENV.fetch('THREE_SIXTY_DIALOG_PARTNER_REST_API_ENDPOINT', 'https://stoplight.io/mocks/360dialog/360dialog-partner-api/24588693')
-      url = URI.parse("#{base_uri}/configs/webhook")
+      base_uri = ENV.fetch('THREE_SIXTY_DIALOG_WHATS_APP_REST_API_ENDPOINT', 'https://stoplight.io/mocks/360dialog/360dialog-partner-api/24588693')
```

Comment:
This does not need to be re-evaluated all the time. It would be best to have this in a central location and not let the server start before it's set. (or disable the relevant part of the code)

Reply:
We had it in a central location in our Settings model, but then we ripped that out. Can you be more specific, or add a commit as to what you are actually proposing?

Reply:
The settings cached was a pain because settings get read from the `ENV` and *persisted* to the database. There is nothing wrong about having `ENV.fetch`es in one place and reading them as early as possible.

I'm just a little concerned that we learn about misconfigurations too late in the code execution.